client/state/data-layer/wpcom-http/pipeline/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import {
+	compact,
+	pick,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { applyDuplicatesHandlers, removeDuplicateGets } from './remove-duplicate-gets';
+
+/**
```

Comment:
although I haven't put any here I felt it was another simple-yet-essential plug into the API like the `shouldAbort` flag. I could easily imagine dispatching some error-reporting mechanism for things like failed network requests. this would necessitate jumping out of the HTTP layer and back into Redux